docs/GettingStarted.md

```diff
@@ -234,7 +233,7 @@ On a defender machine with [eBPF installed](#installing-ebpf-for-windows), do th
 1. Install and set up a DNS server.
 1. Make sure that either test signing was enabled as discussed in
    [Installing eBPF for Windows](#installing-ebpf-for-windows), or the kernel debugger (KD) is attached and running.
-1. Install [clang](https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe)
+1. Install [clang](https://github.com/llvm/llvm-project/releases/download/llvmorg-17.0.3/LLVM-17.0.3-win64.exe)
```

Comment:
Does LLVM 17 work with cilium?  I thought it didn't.

Reply:
Unfortunately, supply chain security trumps this. Folks consuming the Cilium BPF programs we will either need to fix prevail or the BPF programs.

We cannot deploy a production version of our products based on a code base that isn't getting security fixes. No one is signed up to provide security fixes for LLVM 11.0.1, but Microsoft's Visual Studio team is providing security fixes for LLVM 17.0.3 (at least the version that ships with Visual Studio).

Reply:
Sounds like a good reason to me.

Reply:
With a few very minor bug fixes in the Cilium code, it now compiles using LLVM 17!